src/integration.test.ts

```diff
@@ -0,0 +1,22 @@
+import { SkynetClient } from "./client";
+// import { Buffer } from "buffer";
+import { pki } from "node-forge";
+// import { readData } from "./utils";
+
+const { publicKey, privateKey } = pki.ed25519.generateKeyPair();
+const client = new SkynetClient("https://siasky.dev");
+
+const datakey = "HelloWorld";
+const json = { data: "thisistext" };
+
+// skip - used to verify end-to-end flow
+describe.skip("siasky.dev end to end", () => {
+  it("to update the file in the SkyDB", async () => {
+    // set the file in the SkyDB
+    await client.db.setJSON(privateKey, datakey, json);
+
+    // get the file in the SkyDB
+    const actual = await client.db.getJSON(publicKey, datakey);
```

Comment:
Should we be returning the revision number here? I'm not too sure, if the caller cares about it he can just use the registry himself and not go through `setJSON` and `getJSON` which essentially are convenience methods right.

I think `setJSON` and `getJSON` should mirror each other as much as possible and thus only return json.